twitter/entities.go

```diff
@@ -71,3 +72,33 @@ func (i Indices) Start() int {
 func (i Indices) End() int {
 	return i[1]
 }
+
+// VideoSizes contain the different video sizes that are available.
+// https://dev.twitter.com/overview/api/entities#obj-sizes
+type VideoSizes struct {
+	Thumb  VideoSize `json:"thumb"`
+	Large  VideoSize `json:"large"`
+	Medium VideoSize `json:"medium"`
+	Small  VideoSize `json:"small"`
+}
+
+// VideoSize as returned in a VideoSizes.
```

Comment:
This can probably be described a little better as `// VideoSize describes the dimensions of a video media object.`. But this is very minor.